tests/network/vmi_networking.go

```diff
@@ -176,6 +176,8 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 				}
 				Expect(libnet.CheckMacAddress(vmi, "eth0", vmi.Status.Interfaces[0].MAC)).To(Succeed())
 
+				By("check IP/s reported on the VMI matches the ones on the pod")
+				Expect(libnet.ValidateVMIandPodIPMatch(vmi, vmiPod)).To(Succeed(), "Should have matching IP/s between pod and vmi")
 			},
 				Entry("[test_id:1539]the Inbound VirtualMachineInstance", &inboundVMI),
```

Comment:
Please consider stating that the VMI is expected to implicitly have the default binding -
Which is also implicitly assumed to be `bridge` since it is not explicitly set in the KubeVirt CR.
https://github.com/kubevirt/kubevirt/blob/f68fb96c0e70b818b2b0c606aab35f68c9eb462d/pkg/virt-config/virt-config.go#L58

Reply:
It will not be correct for all cases, see bellow entries, there are three different VM configurations.
The first one is a VMI with no network/ifaces configured.

Reply:
Done